epam-promo/components/widgets/Tag.tsx

```diff
@@ -11,4 +11,7 @@ export type TagMods = {
 
 export type TagProps = UuiTagCoreProps & TagMods;
 
-export const Tag = createSkinComponent<UuiTagCoreProps, TagMods>(UuiTag);
+export const Tag = createSkinComponent<UuiTagCoreProps, TagProps>(
```

Comment:
The same issue with first generic as for Badge

Reply:
fixed